container/podman/podman.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Provides global docker information.
+package podman
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"time"
+
+	dockertypes "github.com/docker/docker/api/types"
+	"golang.org/x/net/context"
+
+	"github.com/google/cadvisor/info/v1"
+	"github.com/google/cadvisor/machine"
+)
+
+var podmanTimeout = 10 * time.Second
+
+func defaultContext() context.Context {
+	ctx, _ := context.WithTimeout(context.Background(), podmanTimeout)
+	return ctx
+}
+
+func SetTimeout(timeout time.Duration) {
+	podmanTimeout = timeout
+}
+
+func Status() (v1.PodmanStatus, error) {
+	return StatusWithContext(defaultContext())
+}
+
+func StatusWithContext(ctx context.Context) (v1.PodmanStatus, error) {
```

Comment:
Does this function must be exported? This is more general remark: there is substantial amount of exported function in `podman ` package that do not seem to implement an interface and are not used anywhere outside of the package. Can you unexplored them? I would rather expect them to be exported on case-by-case basis when we clearly see a use case for them.